Documentation/community/roadmap.rst

```diff
@@ -70,7 +70,7 @@ Release Cadence
 ~~~~~~~~~~~~~~~
 
 We aim to make 2 to 3 `point releases`_ per year of Cilium and its core components
-(Hubble, Cilium CLI, Tetragon, etc). We also make patch releases available as
+(Hubble, Cilium CLI, Tetragon, etc.). We also make patch releases available as
```

Comment:
nit: Our docs style actually suggest to avoid latin abbreviations:

https://docs.cilium.io/en/stable/contributing/docs/docsstyle/#don-t-use-latin-abbreviations